src/Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs

```diff
@@ -2812,7 +2812,7 @@ private void GenerateHelperFunctions(TextWriter writer)
 
```

Comment:
@vexx32, your last commit had 3 failures in `PowerShell-CI-linux`
named blocks parsing.Ast Validation: <<begin process end dispose>>.$asts[0].Extent.Text
```powershell
Expected strings to be the same, but they were different.
String lengths are both 25.
Strings differ at index 6.
Expected: 'begin dispose end process'
But was:  'begin process end dispose'
at <ScriptBlock>, /home/vsts/work/1/s/test/tools/Modules/HelpersLanguage/HelpersLanguage.psm1: line 149
149:             It "`$asts[$i].Extent.Text" { $asts[$i].Extent.Text | Should Be $a[$i] }
```
named blocks parsing.Ast Validation: <<begin {} process end dispose {}>>.$asts[0].Extent.Text
```powershell
Expected strings to be the same, but they were different.
String lengths are both 31.
Strings differ at index 9.
Expected: 'begin {} dispose {} end process'
But was:  'begin {} process end dispose {}'
at <ScriptBlock>, /home/vsts/work/1/s/test/tools/Modules/HelpersLanguage/HelpersLanguage.psm1: line 149
149:             It "`$asts[$i].Extent.Text" { $asts[$i].Extent.Text | Should Be $a[$i] }
```
Function Pipeline Behaviour.Output Behaviour on Error States.does not execute End {} if the pipeline is halted during Process {}
```powershell
Expected @('BEGIN', 1, 2), but got @('BEGIN', 'PROCESS 1', 'PROCESS 2').
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Language/Scripting/PipelineBehaviour.Tests.ps1: line 42
42:             } | Select-Object -First 3 | Should -Be @( "BEGIN", 1, 2 )
```